react_app/src/App.scss

```diff
@@ -0,0 +1,181 @@
+  background-color: #fbfbfb;
+  font-smooth: always;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.container {
+  max-width: 1140px;
+  width: 100%;
+  margin: auto;
+}
+
+@mixin styling() {
+  background-color: white;
+  border-color: #ccc;
+  border-radius: 4px;
+  border-style: solid;
+  border-width: 1px;
+  box-shadow: 0 .125rem .25rem rgba(0,0,0,.075) !important;
+}
+
+.creation-modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width:100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.6);
+
+  .modal-main {
+    position:fixed;
+    background: white;
+    width: 80%;
+    height: 30%;
+    top:50%;
+    left:50%;
+    transform: translate(-50%,-50%);
+    border-radius : 10px;
+
+    .cancel {
+      border : none;
+      border-radius : 5px;
+      background-color: #e33440;
+      font-size : 18px;
+      color : #fbfbfb;
+      width : 20%;
+      height : 50px;
+      transition : background-color 0.2s ease-in-out;
+    }
+
+    .cancel:hover {
+      background-color : #c92e39;
+      color : #f5f2f2;
+    }
+
+    .cancel:active {
+      outline: none;
+    }
+
+    .upload-indicator {
+      position: fixed;
+      background : rgba(255, 255, 255, 0.85);
+      align-items: center;
+      justify-content : center;
+      width: 100%;
+      height : 100%;
+      border-radius : 10px;
+    }
+
+    .input-dialog {
+      padding-top : 15px;
+      display : flex;
+      flex-direction : column;
+      justify-content : space-evenly;
+      margin-right : auto;
+
+      .dataset-namer {
+        margin-bottom : 10px;
+        border-radius : 8px;
+        margin-left : 34%;
+        width : 30%;
+        transition: all 0.4s ease-in-out;
+      }
+
+      .dataset-namer:focus {
+        margin-left : 10px;
+        width : 97%;
+      }
+
+      .file-upload-dialog {
+        margin-left : 39%;
+        display : none;
+      }
+
+      .file-label {
+        background-color : #328a3a;
+        color : #d7dfe0;
+        font-size : 20px;
+        font-weight : bold;
+        align-self : center;
+        padding-top : 16px;
+        padding-bottom : 16px;
+        padding-left : 10px;
+        padding-right : 10px;
+        border-radius : 5px;
+      }
+    }
+  }
+}
+
+input {
+  border-color : #4287f5;
+}
+input[type=text] {
+  padding-left : 5px;
+  width : 30%;
+  transition: width 0.4s ease-in-out;
+  height: 40px;
+  border-radius : 4px;
+  font-size : 20px;
+}
+
+input[type=text]:focus {
+  width : 100%;
+}
+
+.form-wrapper {
+  display : block;
+  margin-right: 20%;
+  margin-left: 20%;
+  margin-top : 3%;
+}
+
+.embedding-form {
+  display : flex;
+  justify-content: space-between;
+  margin-bottom : 10px;
+
+  .form-category {
+    display : flex;
+    justify-content : space-evenly;
+
+    .category-title {
+      margin-right: 10px;
+    }
+  }
+}
+
+.display-block {
+  display: block;
+}
+
+.display-none {
+  display: none;
+}
+
+.display-flex {
+  display : flex;
+}
```

Comment:
Please add a newline at the end of the file.

Reply:
Done!